docs/deployment/play-with-docker.md

```diff
@@ -46,6 +46,9 @@ git clone https://github.com/openfaas/faas && \
   ./deploy_stack.sh
 ```
 
+!!! info
+    Basic authentication is now enabled by default for your protection so note down your credentials. If you need to disable it pass the flag --no-auth to the ./deploy_stack.sh command above.
```

Comment:
Consider a slight reword:
_Basic authentication is enabled by default for your protection, so note down your credentials. If you want to disable basic authentication then pass the flag --no-auth to the ./deploy_stack.sh command above._